x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -64,10 +75,12 @@ export function SparkPlot({
     width: compact ? px(unit * 3) : px(unit * 4),
   };
 
+  const SparkelineSeries = hasComparisonSeries ? LineSeries : AreaSeries;
+
   return (
     <EuiFlexGroup gutterSize="m" responsive={false}>
       <EuiFlexItem grow={false}>
-        {!series || series.every((point) => point.y === null) ? (
+        {!series || isEmptyTimeseries(series) ? (
```

Comment:
can `isEmptyTimeseries` handle `!series` as well?

Reply:
It could, but TS still complains about `series` be undefined.

<img width="531" alt="Screen Shot 2021-03-08 at 09 28 49" src="https://user-images.githubusercontent.com/55978943/110334390-b75aec80-7ff0-11eb-8316-e80e210d1ddc.png">